src/libraries/System.Private.CoreLib/src/System/IO/WindowsFileStreamStrategy.cs

```diff
@@ -256,6 +279,7 @@ protected unsafe void SetLengthCore(long value)
             Debug.Assert(value >= 0, "value >= 0");
 
             FileStreamHelpers.SetLength(_fileHandle, _path, value);
```

Comment:
I know this is unrelated to your PR, but I noticed we have a getter method and a setter method for length, but their names don't match.

We should consider renaming the setter:

- `FileStreamHelpers.GetFileLength`
- `FileStreamHelpers.SetLength` => `FileStreamHelpers.SetFileLength`

But not in this PR. I [left a comment in Adam's PR](https://github.com/dotnet/runtime/pull/49750#issuecomment-801576960) so it can be addressed there.